README.md

```diff
@@ -60,6 +60,8 @@ Refer the example [here](https://github.com/aws-samples/eks-anywhere-addons/blob
 
 ## Pre-requisite (Linux/MacOS)
 
+Please use the dev container configuration in the `.devcontainer` folder with [devpod](devpod.sh) or any other dev container environment to create minikube cluster with all required pre-requisites such helm, kubectl and flux for local testing. You can skip over to local testing section if you use dev container environment.
```

Comment:
with all *the* required prerequisites

such *as* helm

skip over to *the* local testing section

Reply:
Fixed. Pls check